.eslintignore

```diff
@@ -1 +1,4 @@
-node_modules
\ No newline at end of file
+node_modules
+jest-setups/jest.setup.js
+jest-setups/jest.setup.windows.js
+__tests__/ProgressViewWindows.test.js
```

Comment:
is it necessary to ignore this file?

Reply:
I removed the test file from this file but both jest-setup files gives me a `'platform' is not defined  no-undef` error even though it's defined in the setup.windows